client/src/locales/en-US.json

```diff
@@ -1811,6 +1811,20 @@
         "text": {
             "loadDocumentError": "Could not load this document.",
             "loadingDocument": "Loading document..."
+        },
+        "players": {
+            "playback": {
+                "title": "Playback speed",
+                "speed": {
+                    "0": "x0.25",
```

Comment:
May as well name the key `0_25` or `2`, so we don't have to ask "What's 'playback.speed.0'?" when reading the code.